.github/workflows/smoke.yml

```diff
@@ -88,6 +88,29 @@ jobs:
     # Fail if any changes were written to source files.
     - run: git diff --exit-code
 
+  smoke-fr:
+    runs-on: ubuntu-latest
+    name: Fraggle Rock
+
+    steps:
+    - name: git clone
+      uses: actions/checkout@v2
+
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+
+    - run: yarn install --frozen-lockfile --network-timeout 1000000
+    - run: yarn build-report
+
+    - run: sudo apt-get install xvfb
+    - name: yarn smoke --fraggle-rock
+      run: xvfb-run --auto-servernum yarn smoke --debug --fraggle-rock -j=1 --retries=2 a11y lantern seo-passing seo-failing csp
```

Comment:
I suppose the burndown in the inverse of this smoke list? It'd be nice to use `--invert` here and list just the tests that are failing (the ones we need to fix).

Reply:
> I suppose the burndown in the inverse of this smoke list? It'd be nice to use --invert here and list just the tests that are failing (the ones we need to fix).

Great call, I'll do that when the list is a smidge shorter :D